pkg/cmd/cli/cmd/cancelbuild.go

```diff
@@ -3,166 +3,238 @@ package cmd
 import (
 	"fmt"
 	"io"
+	"strings"
+	"sync"
+	"time"
 
-	"github.com/golang/glog"
 	"github.com/spf13/cobra"
 	kapi "k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/errors"
+	"k8s.io/kubernetes/pkg/api/meta"
 	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
-	"k8s.io/kubernetes/pkg/kubectl/resource"
+	"k8s.io/kubernetes/pkg/util/wait"
 
 	buildapi "github.com/openshift/origin/pkg/build/api"
+	buildutil "github.com/openshift/origin/pkg/build/util"
+	osclient "github.com/openshift/origin/pkg/client"
+	cmdutil "github.com/openshift/origin/pkg/cmd/util"
 	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
 )
 
 const (
 	cancelBuildLong = `
-Cancels a pending or running build
+Cancel a running, pending, or new builds
```

Comment:
You may want to reword this differently to convey that we support cancelling multiple builds now. "Cancel running, pending, or new builds"